drivers/gpio/gpio_stellaris.c

```diff
@@ -99,38 +102,12 @@ static int gpio_stellaris_configure(struct device *dev, int access_op,
 	sys_set_bit(GPIO_REG_ADDR(base, GPIO_DEN_OFFSET), pin);
```

Comment:
It's probably better to enable the pin only after configuring the input / output mode (even if at present the driver seems to be used on QEMU target only).